_posts/en/newsletters/2023-04-19-newsletter.md

```diff
@@ -0,0 +1,210 @@
+---
+title: 'Bitcoin Optech Newsletter #247'
+permalink: /en/newsletters/2023/04/19/
+name: 2023-04-19-newsletter
+slug: 2023-04-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **RGB update:** Maxim Orlovsky [posted][orlovsky rgb] to the
+  Bitcoin-Dev mailing list with an update on the state of RGB
+  development.  RGB is a protocol that uses Bitcoin transactions to
+  perform state updates in offchain contracts.  A simple example
+  involves the creation and transfer of tokens, although RGB is designed
+  for more purposes than just token transfer.
+
+  - Offchain, Alice creates a contract whose initial state assigns 1,000
+    tokens to a certain UTXO which she controls.
+
+  - Bob wants 400 of the tokens.  Alice gives him a copy of the original
+    contract plus a transaction which spends her UTXO a new output.
+    That output contains a non-public commitment to the new contract
+    state.  The new contract state specifies the distribution of amounts
+    (400 to Bob; 600 back to Alice) and the identifiers for two outputs
+    which will control those amounts.  The security of this token
+    transfer is now equal to that of Alice's Bitcoin transaction, e.g.
```

Comment:
I see later we clarify onchain/offchain, but at this stage in the writeup, its not clear to me if this 400-token-transfer transaction that Alice gives to Bob is/should be published on chain or can be held offline.

Reply:
Not sure what you are talking about specifically. There are bitcoin transactions, but they have no tokens. The tokens are in RGB, but there we have state transitions and they can't be pushed onchain (they always exist in client-side world)